src/sqlite.js

```diff
@@ -0,0 +1,49 @@
+import {require as requireDefault} from "d3-require";
+
+export default async function sqlite(require) {
+  const sql = await require("sql.js@1.5.0/dist/sql-wasm.js");
+  return sql({locateFile: file => `https://cdn.jsdelivr.net/npm/sql.js@1.5.0/dist/${file}`});
+}
+
+export class SQLiteDatabaseClient {
```

Comment:
Is it going to be odd that this database client doesn't extend the worker's DatabaseClient function, prototype-chain-wise, even though it implements the same interface? It only occurred to me because queryRow matches exactly the worker's implementation and could be re-used here without copying.

I was thinking alternatively, this implementation could move to the worker and it could accept a file attachment for the constructor: `DatabaseClient(FileAttachment("db.sqlite"))`. Though, there's a lot of assumption going into that expression since nothing references "sqlite" statically.

Reply:
DatabaseClient isn’t currently part of the standard library, so I’ll need to move parts of it up here to make that work. I’ll take a crack at it.

Reply:
I thought about some more and I don’t think this is worth the effort. It’s not easy to move the DatabaseClient class into the standard library because it hard-codes the known implementations, all of which currently require coordination with the editor (since they are remote databases that require a database connector, not an in-memory local database like SQLite).